manifests/init.pp

```diff
@@ -21,6 +21,7 @@
   $issue_content = undef,
   $issue_net_template = undef,
   $issue_net_content = undef,
+  $windows_motd_title = 'Message of the day',
```

Comment:
Could you also add an `@param` docline above for documentation?

Reply:
@param has been added.